pkg/console/operator/status.go

```diff
@@ -226,12 +188,12 @@ func (c *consoleOperator) ConditionDeploymentAvailable(operatorConfig *operators
 	return operatorConfig
 }
 
-func (c *consoleOperator) ConditionDeploymentNotAvailable(operatorConfig *operatorsv1.Console) *operatorsv1.Console {
+func (c *consoleOperator) ConditionDeploymentNotAvailable(operatorConfig *operatorsv1.Console, message string) *operatorsv1.Console {
```

Comment:
Changed this to take a message string so we can pass in `pods: 0,1,2` which is more informative.